src/crimson/os/seastore/seastore_types.cc

```diff
@@ -92,8 +92,39 @@ std::ostream& operator<<(std::ostream& out, segment_seq_printer_t seq)
   }
 }
 
+namespace details {
```

Comment:
Not really, just want to add a explicit namespace specifier to avoid unexpected usage.

The next static layout commit will add more helper functions and structures.

Reply:
I suppose the anonymous namespace is the standard way to avoid unexpected usage from outside this source file.
https://stackoverflow.com/questions/154469/why-should-you-prefer-unnamed-namespaces-over-static-functions

Anyway, it's a nit.